final/client/src/pages/tracks.js

```diff
@@ -4,7 +4,7 @@ import TrackCard from '../containers/track-card';
 import { Layout, QueryResult } from '../components';
 
 /** TRACKS gql query to retreive all tracks */
-const TRACKS = gql`
+export const TRACKS = gql`
```

Comment:
`_tests_/tracks.tsx` imports this variable but we were never exporting it 🙃

Reply:
Ahh I think this was intentional to match the course content 

https://github.com/apollographql/odyssey-lift-off-part4/pull/15

Because we never showed using it in tests in Lift-off, wanted to get rid of any extraneous details... do we need that test?

Reply:
duplicating the query for now in the test file so we can avoid the disruption in the course